glossary.md

```diff
@@ -18,6 +18,31 @@ higher BCV means a lower discount for bonders and higher inflation by the
 protocol. A lower BCV means a higher discount for bonders and lower inflation by
 the protocol.
 
+### DAO
+
+Decentralized Autonomous Organization, is a governance mechanism for making
+decisions in a more trustless and collaborative way. Voting rights are often
+bound to a governance token. In OlympusDAO the governance token is OHM.
```

Comment:
I suppose the governance token is sOHM since we vote using sOHM right?

Reply:
For simplicity I would argue it is OHM since sOHM is just an implementation detail and we allow stakers to vote as well. I asked in Discord though. Let's see what other people think.

Reply:
Sh4dow said it is sOHM.